From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@guyburstein__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

andynava committed:
Fix conflict from public repo

sethmanheim committed:
Adding info on outage vs. disaster

Commit:
fix link

sethmanheim committed:
Fixes

Commit:
link

Commit:
link

Commit:
link

ggailey777 committed:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr into ignite

ggailey777 committed:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr into andres

seanli1988 committed:
updated limits and Nonconfigurable server parameters

seanli1988 committed:
Nonconfigurable server parameters

Commit:
Updated dynamic group creation docs. New screenhots, fixed insturctions.

sethmanheim committed:
Updates

sethmanheim committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into geo-dr

Commit:
Removed reference to old behavior where the SSGM setting had to be enabled to make DGs work.

sethmanheim committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into geo-dr

sethmanheim committed:
Adding REST API link

sethmanheim committed:
Edits

Commit:
freshness

seanli1988 committed:
removed bad webapp link from the landing page

seanli1988 committed:
removed bad webapp link from the landing page

seanli1988 committed:
removed bad link

Commit:
another new file with screenshots and updated TOC

Commit:
portal

wmgries committed:
Fix broken table in scalability limits.

wmgries committed:
Update Azure File Sync blurb.

wmgries committed:
Azure File Sync release notes.

seanli1988 committed:
updated server parameters

wmgries committed:
Cleaned up and updated the Azure Files FAQ.

wmgries committed:
Replace Azure Files videos with more current content.

wmgries committed:
Iterate last revision date.

wmgries committed:
Remove duplicate troubleshoot.

wmgries committed:
Fix up acrolinx and other issues

wmgries committed:
More fixes.

sethmanheim committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into geo-dr

wmgries committed:
Add excluded files to FAQ, clarify AFS GRS support statement.

wmgries committed:
Revert "Removing encryption at-rest comment per @RenaShahMSFT. Figure out after Ignite."

This reverts commit 09a4fa9b27642e9e4b15272e1e8ca058245669f3.

wmgries committed:
Encryption at rest is now on by default.

TerryLanfear committed:
removing an article

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into acs_rem

wmgries committed:
Clean up FAQ for snapshots, and add (preview) tag after first mention of snapshots.

TerryLanfear committed:
fix per editor

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into acs_rem

Commit:
updates to file before publishing

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fix

Commit:
fixing links

wmgries committed:
Changes for @tamram.

Commit:
adding in image from md section

jeffpatt24 committed:
Update storage-files-faq.md

Added note about change detection

nsoneji committed:
Updated with Scout update 6 release notes

ggailey777 committed:
V2 updates and remove func run

vhorne committed:
DNS forwarder edits

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fix

zhouwangzw committed:
fix BOM header; add links in home.md; add important note for subscription key

wmgries committed:
Merge pull request #4 from jeffpatt24/patch-3

Update storage-files-faq.md

rothja committed:
Updating screenshots for SQL VM for 2017

ggailey777 committed:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr into ignite

sethmanheim committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into geo-dr

ggailey777 committed:
Fixed red box

wmgries committed:
Minor changes based on feedback from @RenaShahMSFT and @jeffpatt24.

wmgries committed:
Temporarily remove release notes per @tamram.

ggailey777 committed:
Link fix

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fix

yujhongmicrosoft committed:
Update documentation-government-functions.md

yujhongmicrosoft committed:
Delete documentation-government-function10.png

yujhongmicrosoft committed:
Delete documentation-government-function11.png

yujhongmicrosoft committed:
Removed subscription IDs

belmaiastar committed:
Version guarantee for application upgrade

belmaiastar committed:
Version guarantee for application upgrade2

curtand committed:
freshness plus a little

Commit:
merge conflict

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fresh1010

Commit:
path

curtand committed:
8 total files

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fresh1010

curtand committed:
last freshness articles

paulshealy1 committed:
PostgreSQL is only installed in CentOS DSVM

aashishb committed:
Updated image create command

Added extra - for the parameter manifest-id in az ml image create command

yujhongmicrosoft committed:
Update documentation-government-functions.md

curtand committed:
fix formatting

curtand committed:
fix formatting

ErikjeMS committed:
changes per Andrew

TerryLanfear committed:
updated master redirect file

v-thepet committed:
fix

dlepow committed:
Incorp'd review comments from Josh Poulson

curtand committed:
chnage date, minor edit

Commit:
24760

TerryLanfear committed:
fix

sethmanheim committed:
Freshness review and updates

rothja committed:
Fixing screen shot and one step

yujhongmicrosoft committed:
Update documentation-government-functions.md

curtand committed:
minor edit

Commit:
bookmark

dlepow committed:
Resolved merge error

ErikjeMS committed:
changes for Heath per Charles

alkohli committed:
Fixed VMware ESXi version to 5.5 and 6.0 only

Based on feedback from Ravi as 6.5 does not seem to work with VMDK.

ErikjeMS committed:
acrolinx

v-kents committed:
corrected link to Bing Entities Search page

Commit:
Update sql-database-metrics-diag-logging.md

ErikjeMS committed:
acrolinx

ErikjeMS committed:
acrolinx

ErikjeMS committed:
acrolinx

wmgries committed:
Fix localization issue.

dlepow committed:
Updated NC steps and screenshot

seanli1988 committed:
merged some edits

johnkemnetz committed:
add schemas

seanli1988 committed:
reverted back index changes

rloutlaw committed:
add callout for JDK version to functions quickstart

seanli1988 committed:
Merge branch 'master' into serverparam

Commit:
New samples article and minor revisions.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aas1010

Commit:
Update sql-database-metrics-diag-logging.md

rloutlaw committed:
fix warning

wmgries committed:
Fixed misspelling of "portal".

Commit:
remove a file and update TOC to publish live

Commit:
added PM as msauthor

parakhj committed:
Updating note for branding edit profile

ChrisGMsft committed:
Update the "reported properties" term 

Updated the term "Reported properties" to "Twin's reported properties" to be clear and consistent with the Cloud-to-device communications guidance article.

PRMerger17 committed:
Merge pull request #25146 from v-thepet/linkies

production bug: fix broken links

Lisaco88 committed:
Merge pull request #25031 from TerryLanfear/acs_rem

Acs rem

PRMerger15 committed:
Merge pull request #25141 from vhorne/fixbranch

DNS forwarder edits

Lisaco88 committed:
Merge pull request #24942 from cynthn/3-10617

Images freshness

Commit:
Merge pull request #25142 from paulshealy1/patch-1

PostgreSQL is only installed in CentOS DSVM

PRMerger13 committed:
Merge pull request #22382 from barlanmsft/patch-2

author changes and spellcheck

ktoliver committed:
Merge pull request #23837 from dlepow/batchaccount

Batch account, low pri VM, and quota updates

PRMerger12 committed:
Merge pull request #25156 from alkohli/patch-147

Fixed VMware ESXi version to 5.5 and 6.0 only

ggailey777 committed:
Revert "Fixed red box"

This reverts commit 93e4d78b48c8a5ad5a12e1250d01b376dc8b295d.

ktoliver committed:
Merge pull request #23779 from craigcaseyMSFT/v-craic0915

new file with screenshots and updated TOC

Commit:
ASA, fix event error in hopping window picture (VSTS #10939538)

Commit:
Merge pull request #25150 from sethmanheim/wk-fresh

Freshness review and updates

PRMerger18 committed:
Merge pull request #25160 from johnkemnetz/patch-1

add schemas

ktoliver committed:
Merge pull request #25108 from rothja/update2017screenshots2

Updating screenshots for SQL VM for 2017

PRMerger14 committed:
Merge pull request #25159 from danimir/patch-1

Update sql-database-metrics-diag-logging.md

PRMerger18 committed:
Merge pull request #25165 from Jim-Parker/update101017

added PM as msauthor

Commit:
Merge pull request #25169 from parakhj/master

Updating note for branding edit profile

Commit:
Merge pull request #25161 from rloutlaw/java-functions-patch

add callout for JDK environment variable to functions quickstart

Commit:
Fixed validation error - I think.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aas1010

Lisaco88 committed:
Merge pull request #25144 from ErikjeMS/master

changes per Andrew

PRMerger13 committed:
Merge pull request #25143 from aashishb/patch-1

Updated image create command

v-thepet committed:
Merge pull request #24982 from wmgries/files-refresh-171009

Azure Files FAQ and File Sync release notes

ErikjeMS committed:
typo fix

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aas1010

PRMerger18 committed:
Merge pull request #25139 from curtand/fresh1010-2

last freshness articles

PRMerger13 committed:
Merge pull request #25130 from linggengmsft/application_version

Application upgrade version guarantee

Commit:
Removed applies to INCLUDE.

Commit:
Merge pull request #25177 from ErikjeMS/master

typo fix

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aas1010

ErikjeMS committed:
changes per Andrew

v-thepet committed:
Merge pull request #24502 from dlepow/fix

VM sizes: Ubuntu RDMA updates

ErikjeMS committed:
changes per Andrew

v-thepet committed:
Merge pull request #25178 from minhe-msft/master

ASA, fix event error in hopping window picture (VSTS #10939538)

ChrisGMsft committed:
Updated "twin" to "device twin"

Per Elio's feedback

ErikjeMS committed:
changes per Andrew

ErikjeMS committed:
Add files via upload

v-thepet committed:
Merge pull request #25170 from Minewiskan/aas1010

Aas1010

ErikjeMS committed:
changes per Andrew

ktoliver committed:
Merge pull request #25138 from curtand/fresh1010

freshness plus a little editing

v-thepet committed:
bogus

v-thepet committed:
deleting article

Commit:
Added whitespace to make TIP format correctly.

cloudbooster committed:
October doc updates

anoobbacker committed:
Updating the script docs

cloudbooster committed:
Oct update,updated based on inputs

anoobbacker committed:
Updates to scripts sections

ggailey777 committed:
Updates with more versioning details

ggailey777 committed:
Merge branch 'andres' of github.com:ggailey777/azure-docs-pr into andres

ggailey777 committed:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr into andres

PRMerger17 committed:
Merge pull request #25187 from anoobbacker/patch-20

Updating the script docs

nsoneji committed:
Updated vCon_MT bits name

nsoneji committed:
minor update and corrected Note format

nsoneji committed:
corrected the table format

dominicbetts committed:
Update name of HTTP to HTTPS

Commit:
freshness + adding an unsupported note for support

Commit:
forgot to add a comma

mumian committed:
update sreenshot

dominicbetts committed:
Bring Acrolinx scores up to bar

dominicbetts committed:
Change HTTP name to HTTPS

mumian committed:
remove the HBase sentiment analysis article

itechedit committed:
Merge pull request #25199 from mumian/iqscreenshot1011

update sreenshot

itechedit committed:
Merge pull request #24881 from piotrci/DGUpdateCreationFlow

Updated Dynamic Group creation flow

ktoliver committed:
Merge pull request #25181 from v-thepet/whoknows

production bug: fix validation warning

bsiva committed:
Updating OS support matrix with Debian

sethmanheim committed:
Updates per feedback

sethmanheim committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into geo-dr

sethmanheim committed:
Adding one more link

ktoliver committed:
Merge pull request #25158 from v-kents/1096320

corrected link to Bing Entities Search page

itechedit committed:
Merge pull request #25189 from ggailey777/andres

Andres PR from public repo

ktoliver committed:
Merge pull request #25197 from dominicbetts/hub-http-1

Update name of HTTP to HTTPS

Commit:
Merge pull request #25172 from ChrisGMsft/patch-1

Update the "reported properties" term

itechedit committed:
Merge pull request #25198 from Blackmist/fresh5

freshness + adding an unsupported note for support

ErikjeMS committed:
fix per Lisa

ErikjeMS committed:
lower case high availability

Lisaco88 committed:
Merge pull request #25201 from mumian/removeTSA1011

remove the HBase sentiment analysis article

nsoneji committed:
Added sequence number for upgrade steps

Commit:
Merge pull request #25205 from bsiva/master

Updating ASR A2A OS support matrix with Debian

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

ktoliver committed:
Merge pull request #25200 from dominicbetts/hub-http-2

Change HTTP to HTTPS where appropriate

Commit:
storage : limits test

v-alje committed:
Merge pull request #25208 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger13 committed:
Merge pull request #25196 from zhouwangzw/zhouwang/update-doc2

fix BOM header; add links in home.md; add important note for subscrip…

PRMerger12 committed:
Merge pull request #25211 from georgewallace/limitsupdate

storage : limits test

rjagiewich committed:
Merge pull request #25180 from ErikjeMS/master

changes per Andrew

ErikjeMS committed:
clarify time server field

ErikjeMS committed:
Update azure-stack-run-powershell-script.md

nsoneji committed:
updated number in intall update components

ggailey777 committed:
Tom review comments

ggailey777 committed:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr into ignite

PRMerger10 committed:
Merge pull request #25213 from ErikjeMS/master

clarify time server field

ggailey777 committed:
Merge branch 'patch-1' of github.com:yoichinak/azure-docs into yoichinak

ggailey777 committed:
Updates to customer addition for SendGrid APIKey

CarlRabeler committed:
cats

nsoneji committed:
removed number from download and marking steps 1 from extracting .zip

v-shils committed:
Merge pull request #25162 from seanli1988/serverparam

Serverparam

ggailey777 committed:
Clarify version setting

CarlRabeler committed:
Update sql-database-single-database-resources.md

mumian committed:
add links

sethmanheim committed:
Addressing issues

mumian committed:
incorporate pm feedback

Commit:
Merge pull request #25226 from mumian/vpnlink1011

add links

rjagiewich committed:
Merge pull request #24702 from sethmanheim/geo-dr

Adding article on Service Bus geo-disaster recovery

Commit:
fixing formatting issues

PRMerger14 committed:
Merge pull request #25222 from ggailey777/yoichinak

Updates to customer addition for SendGrid APIKey

cloudbooster committed:
Updated author with github alisas

Commit:
minor updates

v-shils committed:
Merge pull request #25126 from yujhongmicrosoft/functions

Functions updates

Commit:
fixing date

Lisaco88 committed:
Merge pull request #25070 from cynthn/24760

Fixing links in 24760

v-shils committed:
Merge pull request #25093 from nsoneji/master

 Updated with Scout update 6 release notes

PRMerger19 committed:
Merge pull request #25230 from Blackmist/freshness

minor updates

PRMerger20 committed:
Merge pull request #25227 from mumian/iq1011

incorporate pm feedback

mmacy committed:
Fix merge conflict

gsilva5 committed:
update service updates page

added new link for service updates:  https://azure.microsoft.com/en-us/updates/accelerated-networking-in-expanded-preview/

sanjeev3 committed:
Update ecommerce-retail-catalog-moderation.md

sanjeev3 committed:
Update facebook-post-moderation.md

sanjeev3 committed:
Update ecommerce-retail-catalog-moderation.md

gsilva5 committed:
Update virtual-network-create-vm-accelerated-networking.md

HJToland3 committed:
content tweak

HJToland3 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into localworking

PRMerger13 committed:
Merge pull request #25234 from sanjeev3/master

Added License Terms to Tutorials

v-shils committed:
Merge pull request #25183 from harijayms/imdsoctdocsupdate

Instance Metadata October doc updates

ktoliver committed:
Merge pull request #25223 from CarlRabeler/cats

cats

ggailey777 committed:
Fixed subsccription id

Commit:
Update analysis-services-powershell.md

Commit:
Merge pull request #25232 from mmacy/bureado-patch-1

fix merge conflict in public repo #2557

ktoliver committed:
Merge pull request #25236 from HJToland3/localworking

Localworking

Commit:
Update analysis-services-powershell.md

PRMerger15 committed:
Merge pull request #25233 from gsilva5/patch-8

updated service updates link with new expanded preview link

PRMerger17 committed:
Merge pull request #25239 from Minewiskan/patch-4

Update analysis-services-powershell.md

ktoliver committed:
Merge pull request #25110 from ggailey777/ignite

V2 updates and remove func run
